### DISCUSSION TITLE
What are your performance with Ubiquity EdgeRouter-X ?

### DISCUSSION THREAD

**Discussion by lemassykoi** [Ysurac/openmptcprouter#3036] [Q&A] [answered]
Hi, I installed latest version on an Edgerouter-X, all is working fine, but performance are very poor.
WAN1 is 500 Mbits and WAN2 is 300 Mbits, but bonding give me about 50 Mbits.

I tried with an Oracle OCI and a VM on a remote site, but still poor performances. Next step is to change local MPTCP device.

Thanks.

**Comment by Ysurac** [comment, marked as answer]
Edgerouter-X have a really too slow CPU (all MIPS CPU seems too slow...).
A RPI4 can give up to 550Mb/s with encryption, so for 800Mb/s you need something better than that.
RPI4 doesn't have AES hardware encryption, in this case it should be able to reach 800Mb/s, I will test with Raspberry Pi 5 when I will get it (it have AES hardware encryption and a better CPU).

**Comment by lemassykoi** [comment]
Thanks. It was a spare hardware I had, which was doing nothing. I tried on it because of the small form factor, which is nice
